sql/mob_droplist.sql

```diff
@@ -8800,8 +8800,8 @@ INSERT INTO `mob_droplist` VALUES (1728,0,0,1000,2518,120);
 INSERT INTO `mob_droplist` VALUES (1728,0,0,1000,2521,40);
 INSERT INTO `mob_droplist` VALUES (1728,0,0,1000,4203,100);
 INSERT INTO `mob_droplist` VALUES (1729,0,0,1000,4277,100);
-INSERT INTO `mob_droplist` VALUES (1730,1,1,1000,16935,840);
-INSERT INTO `mob_droplist` VALUES (1730,1,1,1000,16936,140);
+INSERT INTO `mob_droplist` VALUES (1730,1,1,1000,16935,850);
+INSERT INTO `mob_droplist` VALUES (1730,1,1,1000,16936,150);
```

Comment:
Good catch on this not adding to 1000, however the drop rate seems extremely wrong according to ffxidb:
http://ffxidb.com/zones/151/moo-ouzi-the-swiftblade

Should be closer to 920 and 80.